lib/libvcc/vcc_compile.h

```diff
@@ -100,6 +100,15 @@ struct token {
 
 typedef const struct type	*vcc_type_t;
 
+/*
+ * A type attribute is information already existing, requiring no processing
+ * or resource usage.
+ *
+ * A type method call and may do (significant processing, change things, eat
```

Comment:
line 107 reads badly, but I think I can guess what it is meant to say.

Reply:
Bad refactoring of:

https://github.com/varnishcache/varnish-cache/blob/be1191ea9d5b0230ebb6c187ec40a51e23844d83/lib/libvcc/vcc_expr.c#L823-L824

Will fix.